File: Modul 2/Soal 5/PRAK205-2310817210005-Galih_Aji_Sabdaraya.py
import math
A = input().split()
if len (A) == 2:
    A, B = map(int, A)
elif len (A) == 1:
    A = int(A[0])
    B = int(input())
C = int(math.sqrt(B ** 2 - A ** 2))
Keliling = int(A + B + C)
Luas = int(A * C / 2)
print(f"\nAlas = {C} cm")
print(f"Tinggi = {A} cm")
print(f"Keliling = {Keliling} cm")
print(f"Luas = {Luas} cm^2")

File: Modul 2/Soal 5/PRAK205-2310817210005-Galih_Aji_Sabdaraya.c
#include <stdio.h>
#include <math.h>

int main(){
    int A, B;
    scanf("%d %d", &A, &B);
    int C = sqrt((B * B) - (A * A));
    int Keliling = A + B + C;
    int Luas = (A * C)/ 2;
    puts("\n");
    printf("Alas = %d cm\n", C);
    printf("Tinggi = %d cm \n", A);
    printf("Keliling = %d cm\n", Keliling);
    printf("Luas = %d cm^2", Luas);
    
    return 0;
}